keyboards/blank_tehnologii/manibus/keymaps/via/keymap.c

```diff
@@ -0,0 +1,112 @@
+          _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+    //   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+          _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+    //   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+          _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+    //   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+          _______, _______, _______, _______, _______, _______,                         _______, _______, _______, _______, _______, _______,
+    //   ├────────┼────────┼────────┼────────┼────────┼────────┼────────┐     ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+          _______, _______, _______, _______, _______, _______, _______,       _______, _______, _______, _______, _______, _______, _______
+    //   └────────┴────────┴────────┴────────┴────────┴────────┴────────┘     └────────┴────────┴────────┴────────┴────────┴────────┴────────┘
+      )
+
+*/
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  [_QWERTY] = LAYOUT(
+//   ┌────────┬────────┬────────┬────────┬────────┬────────┐                       ┌────────┬────────┬────────┬────────┬────────┬────────┐
+      KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,                            KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                            KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_EQL,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                            KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┤                       ├────────┼────────┼────────┼────────┼────────┼────────┤
+      KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                            KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_BSLS,
+//   ├────────┼────────┼────────┼────────┼────────┼────────┼────────┐     ┌────────┼────────┼────────┼────────┼────────┼────────┼────────┤
+      KC_LCTL, KC_HOME, KC_INS,  KC_DEL,  KC_LGUI, KC_SPC,  KC_LALT,       KC_ENT,  KC_BSPC, MO(FN1),KC_LBRC,KC_RBRC,  KC_END,  KC_GRV
```

Comment:
Same issue as default keymap, this also wont work. It would have to be `MO(_FN1)`.